akka-http-core/src/main/scala/akka/http/scaladsl/settings/ConnectionPoolSettings.scala

```diff
@@ -41,7 +41,8 @@ abstract class ConnectionPoolSettings extends js.ConnectionPoolSettings { self:
   def responseEntitySubscriptionTimeout: Duration
 
   /** The underlying transport used to connect to hosts. By default [[ClientTransport.TCP]] is used. */
-  def transport: ClientTransport
+  @deprecated("Deprecated in favor of connectionSettings.transport", "10.1.0")
+  def transport: ClientTransport = connectionSettings.transport
```

Comment:
It wasn't actually marked `@ApiMayChange` so I had to resurrect and deprecate it properly.

Reply:
good